### DISCUSSION TITLE
Test discussion

### DISCUSSION THREAD

**Discussion by onkarmore26** [Cyfrin/foundry-full-course-cu#2003] [General]
I'm receiving this error while compiling:

```
TypeError: Member "getConversionRate" not found or not visible after argument-dependent lookup in uint256.
  --> contract/FundMe.sol:20:17:
   |
20 |         require(msg.value.getConversionRate() >= minimumUsd, "Didn't send enough ETH");
   |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Here is my code :

```solidity
  require(msg.value.getConversionRate() >= minimumUsd, "Didn't send enough ETH");
       

```

Can someone help me figure out what the issue is ?

**Comment by EngrPips** [comment]
Hello @onkarmore26, did you declare in your contract that you will use the `PriceConverter Library` for the type of `uint256`?

**Reply by onkarmore26** [reply]
I forgot to declare it , now i have included it and it is getting compiled. Thanks for the help.

**Reply by EngrPips** [reply]
You are welcome.
